plugins/modules/consul_token.py

```diff
@@ -237,6 +237,12 @@ class ConsulTokenModule(_ConsulModule):
 
     create_only_fields = {"expiration_ttl"}
 
+    def read_object(self):
+        # if `accessor_id` is not supplied we can only create objects and are not idempotent
+        if not self.params.get(self.unique_identifier):
+            return
```

Comment:
While `return` is equivalent to `return None`, it's easier to understand if you explicitly add `None`:
```suggestion
            return None
```